new-dti-website-redesign/src/app/team/TeamDisplay.tsx

```diff
@@ -0,0 +1,320 @@
+          const isLastBox = index === totalBoxes - 1;
+          const shouldShowCloseButton = showCloseButton && isLastBox;
+
+          return (
+            <div
+              key={index}
+              className={`aspect-square relative border-r border-b border-border-1 ${
+                isLastBox ? 'border-r-0' : ''
+              }`}
+            >
+              {shouldShowCloseButton && onClose && (
+                <IconButton
+                  aria-label="Close member details"
+                  onClick={onClose}
+                  variant="tertiary"
+                  className="absolute w-full h-full border-0 rounded-none"
+                >
+                  <XIcon size={24} />
+                </IconButton>
+              )}
+            </div>
+          );
+        })}
+      </div>
+    );
+  };
+
+  const renderMemberGrid = (
+    members: IdolMember[],
+    options: {
+      isAlumni?: boolean;
+      roleName?: string;
+      selectedMember?: IdolMember;
+      selectedMemberIndex?: number;
+      onMemberClick?: (member: IdolMember, roleName: string) => void;
+      memberDetailsRef?: React.RefObject<HTMLDivElement | null>;
+    } = {}
+  ) => {
+    const columns = getColumnCount();
+    const items = [];
+    const {
+      isAlumni = false,
+      roleName = '',
+      selectedMember,
+      selectedMemberIndex = -1,
+      onMemberClick,
+      memberDetailsRef
+    } = options;
+
+    for (let i = 0; i < members.length; i += 1) {
+      const member = members[i];
+      const isSelected = selectedMember?.netid === member.netid;
+
+      items.push(
+        <div
+          key={member.netid}
+          className={`relative border-r border-border-1 ${(i + 1) % columns === 0 ? 'border-r-0' : ''}`}
+        >
+          {isAlumni ? (
+            <a
+              href={member.linkedin || undefined}
+              target="_blank"
+              rel="noopener noreferrer"
+              className="block"
+            >
+              <MemberCard
+                user={member}
+                image={`/team/teamHeadshots/${member.netid}.jpg`}
+                selected={false}
+              />
+            </a>
+          ) : (
+            <MemberCard
+              user={member}
+              image={`/team/teamHeadshots/${member.netid}.jpg`}
+              selected={isSelected}
+              onClick={() => onMemberClick?.(member, roleName)}
+              className="card-clickable"
+            />
+          )}
+        </div>
+      );
+    }
+
+    // Add filler divs
+    const remainder = members.length % columns;
+    if (remainder !== 0) {
+      const fillers = columns - remainder;
+      for (let i = 0; i < fillers; i += 1) {
+        items.push(
+          <div
+            key={`filler-${isAlumni ? 'alumni-' : ''}${i}`}
+            className={`border-r border-b border-border-1 ${i === fillers - 1 ? 'border-r-0' : ''}`}
+          />
+        );
+      }
+    }
```

Comment:
Good idea for the filler `<div>`s! 

I think we should remove the extra line on the side tho because it looks like the card is empty or didn't load correctly.



<img width="1387" height="872" alt="Image" src="https://github.com/user-attachments/assets/b848207f-280f-4f7e-b61c-560efe4f23bd" />

I think just removing `border-r` should do the trick?

```suggestion
    // Add filler divs
    const remainder = members.length % columns;
    if (remainder !== 0) {
      const fillers = columns - remainder;
      for (let i = 0; i < fillers; i += 1) {
        items.push(
          <div
            key={`filler-${isAlumni ? 'alumni-' : ''}${i}`}
            className={`border-r border-border-1 ${i === fillers - 1 ? 'border-r-0' : ''}`}
          />
        );
      }
    }
```